torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -8820,4 +8821,108 @@ def celu_hook(module, _input, output):
             self.assertEqual(ddp_grads[0], local_model.fc.weight.grad)
             self.assertEqual(ddp_grads[1], local_model.fc.bias.grad)
 
+        def _test_hook_pickling(self, hook, hook_state):
+            learning_rate = 0.01
+            chkpt_file = tempfile.gettempdir() + "/checkpoint.pt"
+            rank = self.rank
+
+            input = torch.randn(1, 1, device=rank)
+            net = torch.nn.Linear(1, 5).to(rank)
+            ddp_model = torch.nn.parallel.DistributedDataParallel(
```

Comment:
nit: 
```suggestion
            ddp_model = DistributedDataParallel(
```
(https://github.com/pytorch/pytorch/blob/8cea5e63def0352a1e72455ec87fec40d3cbab8a/torch/testing/_internal/distributed/distributed_test.py#L47)

Reply:
Good point! I've updated PR according to suggestion.